cli/lib/cli.js

```diff
@@ -56,6 +56,7 @@ const descriptions = {
   detached: 'runs Cypress application in detached mode',
   project: 'path to the project',
   global: 'force Cypress into global mode as if its globally installed',
+  configFile: 'path to a config file to override the default cypress.json. set to "false" to skip specifying a config file.',
```

Comment:
Is this required to be a JSON file? If so, this should be specified in description. Also would opt for use of word 'configuration' over 'config' in description as in previous description. Maybe:

"path to JSON file where configuration values are set. default path is './cypress.json'. pass 'false' to disable."